android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt

```diff
@@ -52,6 +52,34 @@ class BleDidcommModule(private val context: ReactApplicationContext) :
         }
     }
 
+    @ReactMethod
+    fun stopCentral(
+        @Suppress("UNUSED_PARAMETER") options: ReadableMap,
+        promise: Promise
+    ) {
+        try {
+            this.centralManager?.shutdownCentral()
```

Comment:
I'd prefer to throw an error here if the centralManager is not started yet. This is done in other functions as well.

Reply:
As below, I'm happy to follow your suggestion. Idea here was that the outcome is dev wants to have the instance shut down/gone. If we throw an error that needs to be handled but doesn't change the desired outcome.

If you have a (strong) preference I'm happy to make amends

Reply:
As below, I'm happy to follow your suggestion. Idea here was that the outcome is dev wants to have the instance shut down/gone. If we throw an error that needs to be handled but doesn't change the desired outcome.

If you have a (strong) preference I'm happy to make amends